fix displaying of maintenance records containing nonprintables

rbns opened the pull request:
nonprintables are now displayed as escape sequences except for \n
which is kept

Commit:
fix displaying of maintenance records containing nonprintables

nonprintables are now displayed as escape sequences except for \n
which is kept

dra reviewed (changes requested):
after some tests this seems to be fine and sufficient:
record.logfile.gsub("\\n", "\n")

Commit:
replace ansi escapes in machine maintenance presenter

this replaces terminal escape sequences frequently appearing in logfiles.
fixed by @dra.
